exporters/logging/src/main/java/io/opentelemetry/exporters/logging/LoggingMetricExporter.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.exporters.logging;
+
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import java.util.Collection;
+import java.util.logging.Logger;
+
+public class LoggingMetricExporter implements MetricExporter {
+  private static final Logger logger = Logger.getLogger(LoggingMetricExporter.class.getName());
```

Comment:
not sure if we follow the standard of uppercase for static final?

change to LOGGER?

Reply:
The standard says to use upper case for immutable classes, and not sure if Logger is

Reply:
??? citation needed.
What standard is this?

Reply:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Didn't know it also took the object immutability into account.

Reply:
@jkwatson here is the prof that we use that standard :) https://github.com/open-telemetry/opentelemetry-java/blob/master/CONTRIBUTING.md#style-guideline

Reply:
Interesting. Yes, loggers are definitely not side-effect free, so not "immutable" by this definition, I guess? Thanks for the links!